librz/include/rz_cmd.h

```diff
@@ -511,7 +512,7 @@ RZ_API RzCmdStatus rz_cmd_call_parsed_args(RzCmd *cmd, RzCmdParsedArgs *args);
 RZ_API RzCmdDesc *rz_cmd_get_root(RzCmd *cmd);
 RZ_API RzCmdDesc *rz_cmd_get_desc(RzCmd *cmd, const char *cmd_identifier);
 RZ_API RzCmdDesc *rz_cmd_get_desc_best(RzCmd *cmd, const char *cmd_identifier);
-RZ_API char *rz_cmd_get_help(RzCmd *cmd, RzCmdParsedArgs *args, bool use_color);
+RZ_API char *rz_cmd_get_help(RzCmd *cmd, RzConfig *config, RzCmdParsedArgs *args, bool use_color);
```

Comment:
Please add the type annotations here: `RZ_NULLABLE`, `RZ_BORROW` etc. See the `DEVELOPERS.md`

Reply:
yeah true. But please still document the function. There is still a long way to go for documenting everything and incremental improvement makes it at least bearable.

Reply:
There is no documentation :) This is why it would be nice if you add it. Because you just edited the function and probably know best what it does.

> any hint where the documentation should be ?

The doxygen should be added where the function is defined.
Please refer to `DEVELOPERS.md` and read it fully, everything is there explained in detail again.